nix/installer-bootstrap/installer-configuration.nix

```diff
@@ -65,11 +65,15 @@
     pkgs.gptfdisk
     pkgs.parted
     pkgs.cryptsetup
+    pkgs.wget
+    pkgs.curl
+    pkgs.magic-wormhole
   ];
 
-  # save space and compilation time. might revise?
-  hardware.enableAllFirmware = lib.mkForce false;
-  hardware.enableRedistributableFirmware = lib.mkForce false;
+  # add in firmware so usb-ethernet adapters work
```

Comment:
How much size does this add to the ISO? IIRC it was quite a lot. But being able to use an Ethernet adapter is useful, maybe we can just pull out that firmware. What chipset is yours?

Reply:
It adds approximately 500 MB.  Given that the minimum size of a USB flash drive these days is 4 GB, I don't think it's a problem.  (If your concern is the transfer speed, you need to set `bs=1048576` when you invoke `dd`.

The key firmwares to make sure we have are the ones for Realtek RTL815x.  The RTL8152 is inside Apple's gigabit ethernet adapter.

Reply:
It looks like it "only" actually adds 400MB, though there is a lot of junk in there we absolutely will not need. I am mostly concerned about build and download time. I will fiddle with this a bit after accepting the PR, it should not be hard to pull out just the relevant bits from `linux-firmware` for common USB LAN adapters.